conferences/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^(?P<zosia_id>\d+)/register$', views.register, name='user_zosia_register'),
```

Comment:
czy id zosi jest tutaj konieczne? jeżeli zosia jest tylko jedna aktywna to warto, żeby można było można edytować preferencje tylko dla aktywnej zosi :)